### PR TITLE
Added GitHub repo meta.

### PR DESCRIPTION
**Pull request by manwar**
Hi @robrwo 

Please review the PR.

Many Thanks.
Best Regards,
Mohammad S Anwar

### CODE REVIEW

**Comment by robrwo** [comment]
This module hasn't been touched in several years. Do you want to take over maintenance?

**Comment by manwar** [comment]
I am happy to take over the maintenance of this distribution.

Can you please merge the PR first?

After that I will delete the clone of the same. Then you can transfer the ownership of the distribution to me in GItHub.

I would also need the co-maint rights so that I can upload the changes to PAUSE.

File: Makefile.PL [hunk 1]
```diff
@@ -24,4 +24,15 @@ WriteMakefile(
     },
     dist                => { COMPRESS => 'gzip -9f', SUFFIX => 'gz', },
     clean               => { FILES => 'File-Temp-Trace-*' },
+    (eval { ExtUtils::MakeMaker->VERSION(6.46) } ? (META_MERGE => {
```
Comment (robrwo) [inline Makefile.PL:27]: You may also want to consider adding the bugtracker (RT)

**Review by robrwo — APPROVED** [review]

**Comment by robrwo** [comment]
Done. You have contributor perms to this repo and comaint on PAUSE.

**Pull request merged**